### PR TITLE
[TTAHUB-1121] Fix other-entity duplicating objectives and not showing TTA provided on visit

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

A bug was discovered if you create a OE report with multiple entities.

## How to test

1. Create an OE report with multiple entities
2. Add objectives with tta
2. Save and leave the report
3. Revist the report notice there is one objective per entity and TTA is blank

We should test creating an OE with multiple entities taking it all the way to approval. Make sure the Review and Approval screens show correctly.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1121

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/services/activityReports.js [hunk 1]
```diff
@@ -369,6 +368,7 @@ export async function activityReportAndRecipientsById(activityReportId) {
   const arId = parseInt(activityReportId, DECIMAL_BASE);
 
   const goalsAndObjectives = await getGoalsForReport(arId);
+  const objectivesWithoutGoals = await getObjectivesByReportId(arId);
```
Comment (GarrettEHill) [inline src/services/activityReports.js:371]: This is all objectives, with and without goals.

File: src/services/objectives.js [hunk 2]
```diff
@@ -185,6 +186,17 @@ export async function getObjectivesByReportId(reportId) {
...
+          {
+            model: ActivityReport,
+            as: 'activityReport',
+            required: true,
+            attributes: ['id', 'activityRecipientType'],
+            where: {
+              activityRecipientType: 'other-entity',
+            },
+          },
+        ],
```
Comment (GarrettEHill) [inline src/services/objectives.js:199]: This is not needed if you just want to get the otherEntity Objectives. Just check that the grantId is null and otherEntityId is not null on the objective. As the activityRecipientType on the report is likely to be removed soon.

Comment (AdamAdHocTeam) [reply src/services/objectives.js:199]: Good idea! I will update this 😄

Comment (AdamAdHocTeam) [reply src/services/objectives.js]: This has been updated.

**Pull request merged**